dotnet/src/Extensions/PromptTemplates.Handlebars/HandlebarsPromptTemplate.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using HandlebarsDotNet;
+using HandlebarsDotNet.Helpers;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.PromptTemplates.Handlebars.Helpers;
+
+namespace Microsoft.SemanticKernel.PromptTemplates.Handlebars;
+
+/// <summary>
+/// Represents a Handlebars prompt template.
+/// </summary>
+internal class HandlebarsPromptTemplate : IPromptTemplate
```

Comment:
@markwallace-microsoft should this be sealed?

Reply:
Yes, it should be